nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -790,6 +816,80 @@ async fn sha2_hash(file: &mut tokio::fs::File) -> anyhow::Result<ArtifactHash> {
     Ok(ArtifactHash(digest.as_slice().try_into()?))
 }
 
+/// For a given zone, save a zip of its log files into a support bundle path.
```

Comment:
We should document the "path" argument here, as the path to a per-sled directory that will end up zipfile.